jvb/src/main/java/org/jitsi/videobridge/Conference.java

```diff
@@ -1075,13 +1075,34 @@ public Logger getLogger()
         return logger;
     }
 
+    /**
+     * @return {@code true} if this {@link Conference} is in more than one relay mesh.
+     */
+    private boolean inMultipleMeshes()
+    {
+        String meshId = null;
+        for (Relay r : relaysById.values())
+        {
+            if (meshId == null)
+            {
+                meshId = r.getMeshId();
+            }
+            else if (!meshId.equals(r.getMeshId()))
```

Comment:
I'm not sure how `null` should be handled here, but I think it's inconsistent: relay IDs of `[null, "A"]` and `["A", null]` will have different results. Maybe something like `relaysById.values().stream().map(r -> r.getMeshId()).collect(Collectors.toSet()).size() > 1;`?